### DISCUSSION TITLE
Biome as Diagnostics Provider

### DISCUSSION THREAD

**Discussion by thunfisch987** [nvimtools/none-ls.nvim#52] [Ideas]
Biome is not only a formatter but also a linter.
Is there any plan to also include it as linter?
If not, can you help me set it up?

**Comment by l0ngvh** [comment]
Since Biome also support LSP, I think it's gonna be better to use nvim-lspconfig to setup biome

**Reply by thunfisch987** [reply]
I got it working already, but forgot about this Discussion
(I use Lunarvim so i don't know if it would make sense to post the answer here)
